src/ez/DB.java

```diff
@@ -617,7 +617,15 @@ public void addColumn(String table, String column, String columnType) {
 
     Map<String, String> ret = Maps.newLinkedHashMap();
     for (Row row : rows) {
-      String type = new String((byte[]) row.getObject("type"));
+      Object typeObj = row.getObject("type");
+      String type;
+      if (typeObj instanceof byte[]) {
+        type = new String((byte[]) typeObj);
+      } else if (typeObj instanceof String) {
+        type = (String) typeObj;
+      } else {
+        throw new RuntimeException("Unhandled type: " + typeObj.getClass());
+      }
```

Comment:
Arguably, folding a byte[] or String into a String could belong in Ox as a utility.